tests/test_interface.py

```diff
@@ -2193,6 +2196,81 @@ def test_VLanInterfaceIpv6LinkLocalOnly(self, dvs, testlog):
         # one loopback router interface
         assert len(intf_entries) == 1
 
+    def set_loopback_action(self, interface, action):
+        if interface.startswith("PortChannel"):
+            tbl_name = "PORTCHANNEL_INTERFACE"
+        elif interface.startswith("Vlan"):
+            tbl_name = "VLAN_INTERFACE"
+        else:
+            sub_intf_sep_idx = interface.find(VLAN_SUB_INTERFACE_SEPARATOR)
+            if sub_intf_sep_idx != -1:
+                tbl_name = "VLAN_SUB_INTERFACE"
+            else:
+                tbl_name = "INTERFACE"
+
+        fvs = swsscommon.FieldValuePairs([("loopback_action", action)])
+        tbl = swsscommon.Table(self.cdb, tbl_name)
+        tbl.set(interface, fvs)
+        time.sleep(1)
+
+    def loopback_action_test(self, iface):
+        # create interface
+        self.create_l3_intf(iface, "")
+
+        # set interface loopback action in config database
+        action_map = {"drop": "SAI_PACKET_ACTION_DROP", "forward": "SAI_PACKET_ACTION_FORWARD"}
+        action = random.choice(list(action_map.keys()))
```

Comment:
we dont prefer to have randomness in test. Please have coverage for both actions.

Reply:
Done.